pyproject.toml

```diff
@@ -1,3 +1,31 @@
-[tool.black]
-line-length = 79
-include = '''.*\.pyi?$'''
+[tool.ruff]
+lint.select = [
+  # flake8-bugbear
+  "B",
+  # flake8-comprehensions
+  "C4",
+  # pycodestyle
+  "E",
+  # Pyflakes errors
+  "F",
+  # isort
+  "I",
+  # flake8-simplify
+  "SIM",
+  # flake8-tidy-imports
+  "TID",
+  # pyupgrade
+  "UP",
+  # Pyflakes warnings
+  "W",
+]
+lint.ignore = [
+  # flake8-bugbear opinionated rules
+  "B9",
+  # line-too-long
+  "E501",
+  # suppressible-exception
+  "SIM105",
+  # if-else-block-instead-of-if-exp
+  "SIM108",
+]
```

Comment:
I've tried to select a good baseline of rules without being overbearing, disabling a few that are unnecessary.

I didn't include any of Ruff's pylint-derived rules. This repository mentioned pylint but didn't include any configuration, so it's hard to know what was being run. I think the baseline here should be sufficient - many of pylint's default rules have analogies in pycodestyle and pyflakes, which this configuration activates.